schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+            type: object
+            additionalProperties:
+              $ref: '#Example'
+      WithExamples:
+        oneOf:
+          - $ref: '#/$defs/Mixins/$defs/WithSingleExample'
+          - $ref: '#/$defs/Mixins/$defs/WithExampleObjects'
+      WithSchema:
+        properties:
+          schema:
+            $ref: '#Schema'
+          content: false
+      WithSchemaAndExamples:
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/WithSchema'
+          - $ref: '#/$defs/Mixins/$defs/WithExamples'
+      WithContent:
+        properties:
+          schema: false
+          content:
+            type: object
+            additionalProperties:
+              $ref: '#MediaType'
+            minProperties: 1
+            maxProperties: 1
+      WithNameAndLocation:
+        required:
+          - name
+          - in
+        properties:
+          name:
+            type: string
+          in:
+            type: string
+            enum:
+              - path
+              - query
+              - header
+              - cookie
+      WithExplodeAndReserved:
+        properties:
+          explode:
+            type: boolean
+          allowReserved:
+            type: boolean
+            default: false
+      WithEmpty:
+        properties:
+          allowEmptyValue:
+            type: boolean
+            default: false
+      WithExplodeReservedAndEmpty:
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/WithExplodeAndReserved'
+          - $ref: '#/$defs/Mixins/$defs/WithEmpty'
+      StyledSimple:
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/WithExplodeReservedAndEmpty'
+        properties:
+          style:
+            type: string
+            const: simple
+            default: simple
+      StyledMatrix:
+        properties:
+          style:
+            type: string
+            enum:
+              - matrix
+              - label
+              - simple
+            default: simple
+      StyledFormOnly:
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/WithExplodeReservedAndEmpty'
+        properties:
+          style:
+            type: string
+            const: form
+            default: form
+      StyledFormComplexNoDefaultOrEmpty:
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/WithExplodeAndReserved'
+        properties:
+          style:
+            type: string
+            enum:
+              - form
+              - spaceDelimited
+              - pipeDelimited
+              - deepObject
+      StyledFormComplex:
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/StyledFormComplexNoDefaultOrEmpty'
+          - $ref: '#/$defs/Mixins/$defs/WithExplodeReservedAndEmpty'
+        properties:
+          style:
+            default: form
+  Objects:
+    $defs:
+      Reference:
+        $anchor: Reference
+        type: object
+        required:
+          - $ref
+        properties:
+          $ref:
+            type: string
+            format: uri-reference
```

```suggestion
          $ref:
            type: string
            format: uri-reference
          summary:
            type: string
        $ref: '#/$defs/Mixins/$defs/Describable'
        unevaluatedProperties: false
```

as per https://spec.openapis.org/oas/v3.1.0.html#referenceObject - reference objects also allow the 'summary' and 'description' properties, and do not permit any additional properties.